src/plugins/guided_onboarding/public/components/guide_panel_step.tsx

```diff
@@ -94,15 +94,17 @@ export const GuideStep = ({
           <>
             <EuiSpacer size="s" />
             <EuiText size="s" data-test-subj="guidePanelStepDescription">
-              {stepConfig.descriptionList.length === 1 ? (
-                <p>{stepConfig.descriptionList[0]}</p> // If there is only one description, render it as a paragraph
-              ) : (
-                <ul>
-                  {stepConfig.descriptionList.map((description, index) => {
-                    return <li key={`description-${index}`}>{description}</li>;
-                  })}
-                </ul>
-              )}
+              {stepConfig.description && <p>{stepConfig.description}</p>}
```

Comment:
There is a space between the paragraph and the list so I might need to wait until Cindy's [PR](https://github.com/elastic/kibana/pull/144164) is in and use the css to remove the margin.